### PR TITLE
Fix temperature check

### PR DESCRIPTION
**Pull request by fry0815**
> Please send your pull requests the develop branch.
> Don't forget to add the change to CHANGELOG.md.

* Does the pull request solve a **related** issue?
* If so, can you reference the issue?
* What does the pull request accomplish? Use a list if needed.
* If it includes major visual changes please add screenshots.

### CODE REVIEW

**Comment by roramirez** [comment]
Hi @fry0815 

This pull request isn't passed Travis. Please use grunt to check the errors.

File: modules/default/currentweather/currentweather.js [hunk 1]
```diff
@@ -296,7 +296,7 @@ Module.register("currentweather",{
 	 */
 	processWeather: function(data) {
 
-		if (!data || !data.main || !data.main.temp) {
+		if (!data || !data.main || typeof data.main.temp === 'undefined') {
```
Comment (nhubbard) [inline modules/default/currentweather/currentweather.js]: Change these to double quotes to fix the linting errors.

Comment (fry0815) [inline modules/default/currentweather/currentweather.js]: Ok, thank you.

**Comment by fry0815** [comment]
Hi @roramirez 
Thank you :)
It's done.

**Comment by MichMich** [comment]
Hi @fry0815, Thanks for your PR. I've merged it into the develop branch.

**Pull request merged**